samples/queryProfileExplain.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Run query explain (regular query)
+//   description: Run query explain for a standard query and print query explain metrics
+
+async function main() {
+  // [START datastore_query_explain_entity]
+
+  // Imports the Cloud Datastore
+  const {Datastore} = require('@google-cloud/datastore');
+
+  // Instantiate the Datastore
+  const datastore = new Datastore();
+  const q = datastore.createQuery('Task');
+  const [, info] = await datastore.runQuery(q, {
+    explainOptions: {analyze: false},
+  });
+  console.log(`info: ${Object.keys(info.explainMetrics)}`);
```

Comment:
When I run this code, I'm seeing error: 
```
client.ts:32
      console.log(`info: ${Object.keys(info.explainMetrics)}`);
                                  ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at quickstart (/usr/local/google/home/bahaaiman/Documents/cfdb-workspace-01/clients/node/nodejs-datastore-client/client.ts:32:35)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.14.0
```

What am I doing wrong in the code?
```
// Imports the Google Cloud client library
const { Datastore } = require('@google-cloud/datastore');

// Creates a client
const datastore = new Datastore({projectId: "my-project"});

async function quickstart() {
    // The Cloud Datastore key for the new entity
    const taskKey = datastore.key(['Task', 'task1']);

    // Prepares the new entity
    const task = {
        key: taskKey,
        data: {
            Description: 'Buy milk2',
            Done: false,
            Slice: [100, 200, 300]
        },
    };

    // Saves the entity
    await datastore.save(task);
    console.log(`Saved ${task.key.name}: ${task.data.Description}`);

    const query = datastore.createQuery('Task');
    const [entities, info] = await datastore.runQuery(query, {
        explainOptions: {analyze: true},
      });
      for (const entity of entities) {
        console.log(`Entity found: ${entity['description']}`);
      }
      console.log(`info: ${Object.keys(info.explainMetrics)}`);

}
quickstart();
```

Reply:
Checked with @danieljbruce offline. Updating the datastore version to 9.0.0 gets rid of the error